pyuvdata/fhd.py

```diff
@@ -186,9 +186,8 @@ def read_fhd(self, filelist, use_model=False, run_check=True, check_extra=True,
         if flags_file is None:
             raise ValueError('No flags file included in file list')
         if layout_file is None:
-            warnings.warn('No layout file included in file list. '
-                          'Support for FHD data without layout files will be '
-                          'deprecated in version 1.5', DeprecationWarning)
+            warnings.warn('No layout file included in file list, '
```

Comment:
I thought I remember there being discussion that we wanted to drop support for not having `antenna_positions` defined... Is it actually okay?

Reply:
What I'm doing here is only issuing a warning so that the files can still be read, but they won't pass a `check`, so for this to work you'll also need to set `run_check=False`. I wanted to leave that possibility open.

Reply:
Yeah if, heaven forbid, someone wants to read old PAPER data ever again they will need this functionality.

Reply:
yes, but not in fhd.py